src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: local("Avenir"),
+    url("fonts/Avenir-Book.ttf") format("ttf"),
+    url("fonts/Avenir-Heavy.ttf") format("ttf"),
+    url("fonts/Avenir.ttc") format("ttc");
+}
+
+:root {
+  --main-border: 1px solid #e1e7ed;
+  --big-height: 70px;
+  --small-height: 42px;
+  --main-width: 100%;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+form {
+  margin-left: 15px;
+}
+
+.big-bar[data-qa] {
+  display: flex;
+  align-items: center;
+  width: var(--main-width);
+  height: var(--big-height);
+  border: var(--main-border);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.big-bar__search[type="search"][data-qa] {
+  border: none;
+  box-shadow: none;
+  height: 100%;
+  font-size: 16px;
+  font-weight: 400;
+  width: var(--main-width);
+  padding-left: 62px;
+  background: transparent url("images/Search.svg") no-repeat 26px center;
+}
+
+.big-bar__search[type="search"]::placeholder {
+  background: transparent;
+  color: #3d4e61;
+}
+
+.big-bar__search[type="search"]:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.big-bar__search[type="search"]:focus {
+  border-radius: 3px;
+  border: var(--main-border);
+  font-weight: 400;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.small-bar[data-qa] {
+  display: flex;
+  align-items: center;
+  width: var(--main-width);
+  height: var(--small-height);
+  border: var(--main-border);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small-bar__search[type="search"] {
+  border: none;
+  box-shadow: none;
+  height: 100%;
+  font-size: 14px;
+  font-weight: 400;
+  width: var(--main-width);
+  padding-left: 33px;
+  background: transparent url("images/Search.svg") no-repeat 13px center;
+  background-size: 11px;
+}
+
+.small-bar__search[type="search"]::placeholder {
+  background: transparent;
+  color: #3d4e61;
+}
+
+.small-bar__search[type="search"]:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  text-shadow: none;
+}
+
+.small-bar__search[type="search"]:focus {
```

Comment:
why do you need to use selector by attribute here, you dont need it, use class and pseudo class

Reply:
Still not fixed